unit/src/registration-form.ts

```diff
@@ -0,0 +1,44 @@
+import { EmailError, 
+    NameError,
+     PasswordError, 
+     RepeatPasswordError, 
+     emailPattern, 
+     latinLettersAndSpacesPattern, 
+     minPasswordLength, 
+     passwordPattern } from "./constants/constants";
+
+
+export class RegistrationForm {
+
+    constructor(protected name: string, protected email: string, protected password: string, protected repeatPassword: string ) {
+  
+    }
+
+    public submitButton(): string {
```

Comment:
Please name the method according to its content: `submitForm()`.

Reply:
done